packages/module/patternfly-docs/pages/index.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Just FYI this follows what was done in the first PR towards JSX Transforms in the React repo (for patternfly-docs files: https://github.com/patternfly/patternfly-react/pull/11524/files#diff-a0d494770302680afb195b3ae0f6ad77b355a330bc58e61664897647ac261bd9